collectors/apps.plugin/apps_plugin.c

```diff
@@ -3601,12 +3609,17 @@ static void send_collected_data_to_netdata(struct target *root, const char *type
     }
     send_END();
 
+
+    kernel_uint_t usedfdpercentage = (kernel_uint_t) ((currentmaxfds * 100) / sysconf(_SC_OPEN_MAX));
+    const char *fdname = "fdperc";
```

Comment:
You don't need this variable anymore. Please delete it.

Reply:
I think github just bugged out, if you mentioned the comment, I removed it in a previous commit!